kafka/signed_blinded_token_issuer_handler.go

```diff
@@ -44,7 +45,13 @@ func SignedBlindedTokenIssuerHandler(
 			continue
 		}
 
-		issuer, appErr := server.GetLatestIssuer(request.Issuer_type, int(request.Issuer_cohort))
+		// check to see if issuer cohort will overflow
+		if request.Issuer_cohort > math.MaxInt16 || request.Issuer_cohort < math.MinInt16 {
+			logger.Error().Msgf("request %s: invalid cohort", blindedTokenRequestSet.Request_id)
+			continue
```

Comment:
WIll need to add add the issuer error like below to avoid empty response 

blindedTokenResults, avroSchema.SigningResult{
					Signed_tokens:     nil,
					Issuer_public_key: "",
					Status:            issuerError,
					Associated_data:   request.Associated_data,
				}